app/javascript/dashboard/components-next/Contacts/ContactsHeader/ContactHeader.vue

```diff
@@ -0,0 +1,106 @@
+<script setup>
+import Button from 'dashboard/components-next/button/Button.vue';
+import Input from 'dashboard/components-next/input/Input.vue';
+import Icon from 'dashboard/components-next/icon/Icon.vue';
+import ContactSortMenu from './components/ContactSortMenu.vue';
+import ContactMoreActions from './components/ContactMoreActions.vue';
+
+defineProps({
+  isEmptyState: {
```

Comment:
Let’s not combine the logic for the list and the view here. Instead, let’s introduce a generic `showSearch` flag and handle the logic on whether to show the search on the parent component.